docs/pages/guides/google-authentication.mdx

```diff
@@ -60,17 +60,19 @@ For Android, install the [`expo-application`](/versions/latest/sdk/application/)
 
 Create a [`androidClientId`](/versions/latest/sdk/auth-session/#googleauthrequestconfig--properties) by following the steps below from the Google Cloud console:
 
-- In the Google Cloud console, go to **Credentials** > **Create OAuth client** page.
+- In the Google Cloud console, go to **Credentials** page and then click **Create credentials** > **OAuth Client Id**.
 - For **Application type**, select **Android** from the dropdown menu.
 - For **Name**, enter the name such as **Android Client ID** to identify different client IDs in case you have more than one for multiple platforms.
 - Under **Package name**, enter the package name of your app. This is the same package name you have set up in the app config file under [`android.package`](/versions/latest/config/app/#package) field, for example, `com.example.myapp`.
 - Under **SHA-1 certificate fingerprint**, you'll need to add the signing certificate value. To get this value:
+  > **warning** If you have an existing app or have uploaded a release build to the Google Play Store, you do not need to re-generate the keystore as described below. You can find the SHA-1 certificate fingerprint in the Google Play Console under **Release** > **Setup** > **App Integrity** > **App signing key certificate** and use that signing certificate value instead.
```

Comment:
I don't think improving wording or adding warning in this section can help. Plus this instruction still does not handle all the cases
- those steps do not cover if keystore was generated but nothing was yet uploaded to play console.
- technically this is incorrect "App signing key certificate" fingerprint is a fingerprint of an app in store. fingerprint from a keystore is a fingerprint of an app before upload, but those instructions treat them as alternatives to each other. 
  - If new project would follow those instructions, it would work when they are testing APK build on EAS but it wouldn't work on production.
  - If existing project would follow those instructions it would work on production but not directlly on APK build by them.
  
I don't think there is a point to handle a case when app was not uploaded yet, at that point play store cert is not know yet, so there is not much point in just handling local case and setting up production latter.

I would suggest writing sth like this(of course with better wording that in my example):

- (At the beginning of this instructions) if this is an app that was never uploaded to the Play console then build and upload sth to play console. It's not necessary publishing to store, just upload sth.
- (In the last point about sha-1)
  - Under **SHA-1 certificate fingerprint**, you'll need to add the fingerprints of the certificates that were use to sign the app. There are 2 values that you need to provide:
    - Fingerprint of a production app downloaded from the play store. You can find the SHA-1 certificate fingerprint in the Google Play Console under **Release** > **Setup** > **App Integrity** > **App signing key certificate**
    - Fingerprint of an APK you built (on your own device or using EAS Build). You can find the SHA-1 certificate fingerprint in the Google Play Console under **Release** > **Setup** > **App Integrity** > **Upload key certificate**
      You could also mentioned here that this value can be extracted from the keystore that we manage, but I think it would be unnecessarily confusing.